[CHEC-1475] Add missing docs (#155)
chec/commerce.js: docs/04_Checkout.md

    <p>Refer to the full response for the "Check Shipping Method <a href="/docs/api/?shell#check-shipping-method">here</a>.</p>
</div>

---

> ## Check is free

Reviewer: ```suggestion
## Check if checkout is free
```

Not sure exactly what the best title here is, but "Check is free" doesn't seem right to me

The `isFree()` method uses `GET /v1/checkouts/{checkout_token_id}/check/is_free` to determine whether the provided

> checkout token's order has a zero payable balance.

Reviewer: ```suggestion
checkout token has a zero payable balance.
```

| -------------------- | ----------- |
| `getShippingOptions(token, data)`  | Gets the available shipping options |

---

## Set tax zone

The `setTaxZone()` helper method uses `GET v1/checkouts/{checkout_token_id}/helper/set_tax_zone` to set the tax zone for

> the provided checkout token's order, either automatically from a provided IP address, or by the geographic data provided

Reviewer: ```suggestion
the provided checkout token, either automatically from a provided IP address, or by the geographic data provided
```

| `setTaxZone(identifier, data)`  | Sets the geographic zone for tax calculation  |

<div class="highlight highlight--note">
  <span>Note</span>
  <p>Refer to the full response for setting the tax zone <a href="/docs/api/?shell#set-tax-zone">here</a>.</p>
</div>

---

## Get location from IP

The `getLocationFromIP()` helper method uses `GET /v1/checkouts/{checkout_token_id}/helper/location_from_ip` to
determine the buyer's physical location based on their IP address. This can be used to automatically select an

> appropriate shipping method during checkout. An IP address can either be provided to be resolved, or it will be detected

Reviewer: ```suggestion
appropriate shipping method during checkout. An IP address can either be provided, or it will be detected
```

automatically. Please note that if the endpoint is called from a server-side request, the resolved IP will be that of
your web server. In this case you must provide the IP address of the customer yourself.

Example request using Commerce.js:

```js
import Commerce from '@chec/commerce.js';

const commerce = new Commerce('{your_public_key}');

commerce.checkout.getLocationFromIP('chkt_L5z3kmQpdpkGlA', '123.45.67.89').then((address) => console.log(address));
```

| Method | Description |
| -------------------- | ----------- |

> | `getLocationFromIP(token, ipAddress)`  | Gets a location from the provided (or your own) IP address |

Reviewer: ```suggestion
| `getLocationFromIP(token, ipAddress = '')`  | Gets a location from the provided (or your own) IP address |
```
